app/src/app/api/v0/city/[city]/inventory/[year]/route.ts

```diff
@@ -18,9 +20,76 @@ export const GET = apiHandler(async (_req: NextRequest, { params }) => {
   }
   inventory.city = city;
 
-  return NextResponse.json({ data: inventory });
+  let body:Buffer|null = null;
+  let headers:Record<string, string>|null = null;
+
+  switch (_req.nextUrl.searchParams.get('format')?.toLowerCase()) {
```

Comment:
Since you're now using the `_req` argument, let's rename it to `req` (_ indicates it is unused and prevents a linter warning about unused arguments).